Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM kaazing/gateway:latest
```

Comment:
Where should the Dockerfiles be placed?

We have a Dockerfile/docker-compose located wthin tools.docker which is responsible with starting up the gateway and the metrics publisher and publishing the data to StatsD/Graphite, in the following forks:
- https://github.com/alingherasim/tools.docker/tree/develop/metrics (forked from kaazing/tools.docker)
- https://github.com/irina-mitrea-luxoft/tools.docker/tree/develop/metrics (forked from alingherasim/tools.docker)

My question is posted due to the fact that the metrics publisher Dockerfile is placed under tools.docker (not yet merged into the main repo though), while the metrics viewer Dockerfile is placed under the metrics project. Should the metrics publisher Dockerfile also be placed under the metrics project for consistency?